File: 0070-climbing-stairs/0070-climbing-stairs.cs
public class Solution {
    public int ClimbStairs(int n) {
        if (n<=2)
        {
            return n;
        }
        List<int> list = new List<int>() {0,1,2};
        for(int i =3; i<=n;i++)
        {
            list.Add(list[i-1] + list[i-2]);
        }
        return list[n];

    }
}

File: 0070-climbing-stairs/0070-climbing-stairs.py
class Solution(object):
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        if n<= 2:
            return n
        listSteps = [0,1,2]
        for i in range (3, n+1):
            listSteps.append(listSteps[i-1] + listSteps[i-2])
        return listSteps[n]
        